docs/tools/vdb_table/data/kdbai.json

```diff
@@ -67,17 +67,17 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "embeddings_image": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "embeddings_structured": {
-    "support": "",
+    "support": "full",
```

Comment:
this attribute is to do with support for graph embeddings, is that something KDB.AI has?